Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.cs

```diff
@@ -352,9 +354,53 @@ public async Task LoadImageFromFile(StorageFile imageFile)
         /// Gets the cropped image.
         /// </summary>
         /// <returns>The cropped image.</returns>
-        public async Task<WriteableBitmap> GetCroppedBitmapAsync()
+        public async Task<WriteableBitmap> GetCroppedImageAsync()
         {
-            return await GetCroppedBitmapAsync(Source, _currentCroppedRect);
+            if (Source == null)
+            {
+                return null;
+            }
+
+            var writeableBitmap = new WriteableBitmap((int)Math.Floor(_currentCroppedRect.Width), (int)Math.Floor(_currentCroppedRect.Height));
+            using (var randomAccessStream = new InMemoryRandomAccessStream())
+            {
+                var bitmapEncoder = await BitmapEncoder.CreateAsync(BitmapEncoder.PngEncoderId, randomAccessStream);
+                await CropImageAsync(Source, _currentCroppedRect, bitmapEncoder);
+                writeableBitmap.SetSource(randomAccessStream);
+            }
+
+            return writeableBitmap;
+        }
+
+        /// <summary>
+        /// Saves the cropped image to a file with the specified format.
+        /// </summary>
+        /// <param name="file">The target file.</param>
+        /// <param name="bitmapFileFormat">the specified format.</param>
+        /// <returns>Task</returns>
+        public async Task SaveAsync(StorageFile file, BitmapFileFormat bitmapFileFormat)
```

Comment:
Remove this which can be accomplished easily enough through the new SaveAsync method

Reply:
I think it can be removed, but should we provide more ways to make it easier to use?